install/kubernetes/helm/subcharts/pilot/values.yaml

```diff
@@ -21,3 +21,10 @@ env:
 cpu:
   targetAverageUtilization: 80
 nodeSelector: {}
+# SDS related options
```

Comment:
Don't we have another sds section, including enabled, etc ? Where did this come from ?

Reply:
yep better to move to https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/values-istio-sds-auth.yaml#L13

Reply:
Moved to install/kubernetes/helm/istio/values-istio-example-sds-token-vault.yaml.